internal/server/data/migrations.go

```diff
@@ -37,6 +37,16 @@ func migrations() []*migrator.Migration {
 				return nil
 			},
 		},
+		// Add roles and migrations to destination table
+		//{
+		//	ID: "2022-05-04T12:13",
```

Comment:
Previously this was handled by `AutoMigrate`

Reply:
This commented out section is something on your TODO?

Reply:
Ya, this is what I need to get working